lib/dor/datastreams/workflow_ds.rb

```diff
@@ -24,7 +23,7 @@ def get_workflow(wf, repo = 'dor')
       Workflow::Document.new(xml.to_s)
     end
 
-    alias_method :[], :get_workflow
+    alias :[] :get_workflow
```

Comment:
I'd like to understand this better.  I've read recently that one or the other is more problematic.  From my recent reading, I concluded that `alias_method` is the way to go, but I'm not entirely clear on why one or the other is better.  For example, http://stackoverflow.com/questions/4763121/should-i-use-alias-or-alias-method